python/requirements/lint-requirements.txt

```diff
@@ -1,16 +1,20 @@
 clang-format==12.0.1
 docutils
-flake8==3.9.1
-flake8-comprehensions==3.10.1
-flake8-quotes==2.0.0
-flake8-bugbear==21.9.2
 mypy==1.7.0
 mypy-extensions==1.0.0
 types-PyYAML==6.0.12.2
 black==22.10.0
-isort==5.10.1
+isort==5.13.2
 semgrep==1.32.0
 shellcheck-py==0.7.1.1
 yq
 types-pycurl==7.45.3.20240421
 types-requests==2.31.0.6
+
+# Python 3.8 dependencies
```

Comment:
we should not run on python 3.8 any more, like anywhere, and this should not keep any python 3.8 dependency.

the base python version used on dev environments and CI is python 3.9. while you can try to support python 3.12, you should not have any requirement statement related to python 3.8 or below.

Reply:
No problem, I will remove it.

Reply:
@aslonnie  I would like to confirm with you:
Since the lint format process on Buildkite uses Python 3.8 to run flake8, how can we avoid issues related to this?